2020-01-Winter/1_linked_lists/list-palindrome/README.md

```diff
@@ -18,16 +23,22 @@ public class question
         //Your code here
     }    
 }
+```
 
-Solution:
+**Solution:**
 
 A simple solution is to use a stack of list nodes.
+
 This mainly involves three steps:
+
 1.Traverse the given list from head to tail and push every visited node to stack.
+
 2.Traverse the list again. For every visited node, pop a node from stack and 
 compare data of popped node with currently visited node.
+
 3.If all nodes matched, then return true, else false.
```

Comment:
The numbered list won't get rendered as such without the space after the numbers. (Tricky, I know.)
```suggestion
3. If all nodes matched, then return true, else false.
```